modules/imgproc/perf/opencl/perf_gftt.cpp

```diff
@@ -72,14 +72,16 @@ OCL_PERF_TEST_P(GoodFeaturesToTrackFixture, GoodFeaturesToTrack,
     checkDeviceMaxMemoryAllocSize(img.size(), img.type());
 
     UMat src(img.size(), img.type()), dst(1, maxCorners, CV_32FC2);
+    std::vector<float> values;
     img.copyTo(src);
 
     declare.in(src, WARMUP_READ).out(dst);
 
     OCL_TEST_CYCLE() cv::goodFeaturesToTrack(src, dst, maxCorners, qualityLevel,
-                                             minDistance, noArray(), 3, 3, harrisDetector, 0.04);
+                                             minDistance, noArray(), 3, 3, harrisDetector, 0.04, values);
 
     SANITY_CHECK(dst);
+    SANITY_CHECK(values, 2.f * FLT_EPSILON, ERROR_RELATIVE);
```

Comment:
I propose to use absolute error 1e-6. It's small enough relatively to values and should cover floating computation issue on different OpenCL devices.

Reply:
Switched the tests back to comparison by absolute error.